Node/ChildProcess.fs

```diff
@@ -35,7 +35,7 @@ type ExecOptions = {
     /// Timeout, in milliseconds. Default is zero (no timeout).
     timeout: int option
 }
-    with static member Default = { encoding = None; cwd = None; timeout = None; maxBuffer = None }
+    with static member Default = { encoding = None; cwd = None; timeout = None }
```

Comment:
Hmm, that won't be different than using createEmpty. I'd rather keep the "only interfaces in the bindings" rules for simplicity for the moment. We can think of a better solution later. 

We can also create a Fable. Helpers.Node package and add this kind of things.